pkg/util/util.go

```diff
@@ -78,6 +78,16 @@ func GenerateRandomString(n int) string {
 	return string(b)
 }
 
+// In checks if the value is in the array
+func In(arr []string, value string) bool {
```

Comment:
Just a thought, since this is in common utils file, should we call this IsStrInSlice instead?

Reply:
The argument does makes sure that its a string slice, do we wanna make it more verbose?

Reply:
Very vague ;)

What if I want to add a function that checks the similar for a different data type you have reserved `In` for strings here ;) But feel free to ignore the comment

Reply:
I can change this in the next PR? this will delay the merge by 50min. I will document this in a follow up issue

Reply:
tracked https://github.com/openshift/odo/issues/1530